translations/Vietnamese/main.ts

```diff
@@ -0,0 +1,179 @@
+		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "",
+		"You are blocking private messages right now.": "Bạn đang chặn tin nhắn riêng.",
+		"You are blocking challenges right now.": "Bạn đang chặn thách đấu.",
+
+		"Your message contained banned words in this room.": "Tin nhắn của bạn chứa từ cấm của phòng.",
+		"You can't send the same message again so soon.": "Bạn không thể gửi tin nhắn cùng một nội dung ngay.",
+		"Due to this room being a high traffic room, your message must contain at least two letters.": "Phòng này đông, tin nhắn tối thiểu 2 từ.",
+
+		"You are already blocking private messages! To unblock, use /unblockpms": "Bạn đã chặn tin nhắn riêng, /unblockpms đ gỡ chặn",
+		"You are now blocking private messages, except from staff and ${rank}.": "Bạn đang chặn tin nhắn riêng, trừ nhân viên và người chơi hạng ${rank}.",
+		"You are now blocking private messages, except from staff and ${status} users.": "Bạn đang chặn tin nhắn riêng, trừ nhân viên và người chơi ${status}.",
+		"You are now blocking private messages, except from staff.": "Bạn đang chặn tin nhắn riêng, trừ nhân viên.",
+		"You are not blocking private messages! To block, use /blockpms": "Bạn không còn chặn tin nhắn riêng nữa, để chặn dùng /blockpms",
+		"You are no longer blocking private messages.": "Bạn không còn chặn tin nhắn riêng nữa.",
+		"You are now blocking all incoming challenge requests.": "Bạn giờ đã chặn mọi thách đấu.",
+		"You are already blocking challenges!": "Bạn đã chặn mọi thách đấu rồi!",
+		"You are already available for challenges!": "Bạn có thể được thách đấu rồi!",
+		"You are available for challenges from now on.": "Bạn đã sẵn sàng với thách thức từ bây giờ.",
+		"You are now blocking challenges, except from staff and ${rank}.": "Bạn đã chặn thách thức, trừ staff và hạng ${rank}.",
+		"You are now blocking challenges, except from staff and ${status} users.": "Bạn đã chặn thách thức, trừ staff và người chơi ${status}.",
+
+		"Staff FAQ": "Hỏi đáp",
+		"You cannot broadcast all FAQs at once.": "Bạn không thể thông báo hết Hỏi Đáp một lần.",
+		"A user is autoconfirmed when they have won at least one rated battle and have been registered for one week or longer. In order to prevent spamming and trolling, most chatrooms only allow autoconfirmed users to chat. If you are not autoconfirmed, you can politely PM a staff member (staff have %, @, or # in front of their username) in the room you would like to chat and ask them to disable modchat. However, staff are not obligated to disable modchat.":			"Người chơi được tự duyệt khi thắng ít nhất một trận và được đăng ký 1 tuần hay lâu hơn. Để chống Spam và troll, mọi phòng chat chỉ cho phép người chơi đã được tự động duyệt tham gia chat. Bạn không tự động duyệt được có thể xin phép nhân viên(Họ có % @ hay # trước tên) trong phòng bạn muốn tham gia chat và xin phép họ tắt chức năng lọc. Tuy nhiên nhân viên không có nghĩa vụ làm điều này nhé.",
```

Comment:
```suggestion
		"A user is autoconfirmed when they have won at least one rated battle and have been registered for one week or longer. In order to prevent spamming and trolling, most chatrooms only allow autoconfirmed users to chat. If you are not autoconfirmed, you can politely PM a staff member (staff have %, @, or # in front of their username) in the room you would like to chat and ask them to disable modchat. However, staff are not obligated to disable modchat.": "Người chơi được tự duyệt khi thắng ít nhất một trận và được đăng ký 1 tuần hay lâu hơn. Để chống Spam và troll, mọi phòng chat chỉ cho phép người chơi đã được tự động duyệt tham gia chat. Bạn không tự động duyệt được có thể xin phép nhân viên(Họ có % @ hay # trước tên) trong phòng bạn muốn tham gia chat và xin phép họ tắt chức năng lọc. Tuy nhiên nhân viên không có nghĩa vụ làm điều này nhé.",
```